### ISSUE TITLE
FFmpegFrameRecorder setVideoCodecName not working

### ISSUE THREAD

**Issue by The-Crocop** [bytedeco/javacv#229]
Hi to me it seems like the setVideoCodecName() Command isnt doing anything. Either I dont know how to use it correctly or there is a bug. Im trying with "H264" , "libx264", "h264". But it always sets the default codec for the format instead. Using  setVideoCodec directly however works without problems.

Can you tell me what are legit Strings to pass to the setVideoCodecName function?

**Comment by saudet** [comment]
There was indeed a little bug there. Fixed in the commit above. Thanks for reporting!

As for how to get a list of available codecs, we can do something like this:
```java
        AVCodec c = av_codec_next(null);
        while (c != null) {
            System.out.println(c.name().getString());
            c = av_codec_next(c);
        }
```
It's all in FFmpeg's API docs...

**Comment by The-Crocop** [comment]
I think there is the same issue with setAudioCodecName!

**Comment by saudet** [comment]
You're right! Thanks, fixed that too.

**Comment by saudet** [comment]
Fix included in version 1.1. Thanks for reporting!
